targets/l2_switch/CMakeLists.txt

```diff
@@ -0,0 +1,43 @@
+# Include directories
+include_directories(
+  ${CMAKE_CURRENT_SOURCE_DIR}
```

Comment:
Do you need to include the current source dir?
Applies to all the other target CMakeLists.txt files.